src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC, useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const peopleFromServer = await getPeople();
+
+        setPeople(peopleFromServer);
+      } catch {
```

Comment:
In the `catch` block, you should also catch a possible `error`, because now the error does not look informative, for example, you can do this:
```
catch(error) { setIsError(`Something went wrong: ${error.message}`)}
```